docs/src/network.md

```diff
@@ -71,6 +71,19 @@ for the entire browser, or for each browser context individually.
 You can optionally specify username and password for HTTP(S) proxy, you can also specify hosts to
 bypass proxy for.
 
+:::note
+Chromium, unlike WebKit and Firefox, will **not** proxy localhost, link locals
+(i.e. 169.254/16), and loopback addresses by default (unless
+running in headful mode on Linux).
+
+If you need these requests (e.g. `page.goto("http://localhost:3030")`) to route
+through the configured proxy, add the special token `<-loopback>` to the `bypass`
+option.
+
+Do not pass `<-loopback>` to non-Chromium browsers as behavior will be undefined
```

Comment:
I added this note since if you remove the following guard, you get a WK fail on macOS headless: https://github.com/microsoft/playwright/pull/8245/files#diff-6abdc33c61672400fa977184bbc3bc7f7ec0ea6873cc6e9d52e93a241c186633R122

So you must conditionally include the special bypass token. (I know the token is Chrome-specific, but I wish we didn't have to branch logic.)